subprojects/docs/src/snippets/dependencyManagement/catalogs-settings/groovy/build.gradle

```diff
@@ -36,6 +36,12 @@ checkstyle {
 }
 // end::use_version[]
 
+// tag::use_version_asprovider[]
+// getting the version of 'mylibrary' requires `asProvider()` when there are other 'mylibrary-...' versions:
+logger.info("Using mylibrary version: " + libs.versions.mylibrary.asProvider().get())
```

Comment:
It would be good to find a more realistic example, with usage outside of a logging statement.
The scala plugin could be used by having the Zinc version and some child version being a compiler plugin version.
The TOML file would contain both versions and a library entry for the compiler plugin.

Reply:
Done, I've changed the snippets. Please take a look.